src/client/app/containers/ChartLinkContainer.ts

```diff
@@ -31,7 +31,7 @@ function mapStateToProps(state: State) {
 			linkText += `&barStacking=${state.graph.barStacking}`;
 			break;
 		case 'line':
-			linkText += `&serverRange=${state.graph.timeInterval.toString()}`;
+			// linkText += `&serverRange=${state.graph.timeInterval.toString()}`;
```

Comment:
I think the commented out code should be removed and a comment added that these is non code for this case but left so show what is going on. (You can leave the next two lines if you want since already there.)